includes/ucf-jobs-feed.php

```diff
@@ -15,19 +15,14 @@
  **/
 function retrieve_job_listing_data( $args ) {
 	$feed_url = get_option( UCF_MAIN_SITE_UTILITIES__CUSTOMIZER_PREFIX . 'jobs_feed_url' );
-	$post_args = array(
-		'body' => wp_json_encode( array(
-			// 'appliedFacets' => {}, // TODO: Add feature for defining job type filters
-			'limit'      => $args['limit'],
-			'offset'     => $args['offset'],
-			'searchText' => ''
-		) ),
-		'headers' => array(
-			'content-type' => 'application/json'
-		)
-	);
 
-	$response      = wp_remote_post( $feed_url, $post_args );
+	$feed_url = add_query_arg( array(
+		'limit'      => $args['limit'],
+		'offset'     => $args['offset'],
+		'searchText' => ''
+	), $feed_url );
+
+	$response = wp_remote_get($feed_url);
```

Comment:
```suggestion
	$response = wp_remote_get( $feed_url );
```

Style guide padding.